lib/WWW/YouTube/Download.pm

```diff
@@ -168,7 +168,7 @@ sub _fetch_title {
 sub _fetch_user {
     my ($self, $content) = @_;
 
-	if( $content =~ /<span class="yt-user-name [^>]+>([^<]+)<\/span>/ ){
+	if( $content =~ /","author":"([^"]+)","/ ){
```

Comment:
It looks like this change broke `https://github.com/xaicron/p5-www-youtube-download/blob/master/t/video_user.t#L14`